versions.txt

```diff
@@ -15,3 +15,5 @@ grpc-google-iam-v2beta:1.9.3:1.9.4-SNAPSHOT
 google-iam-policy:1.9.3:1.9.4-SNAPSHOT
 proto-google-iam-v2:1.9.3:1.9.4-SNAPSHOT
 grpc-google-iam-v2:1.9.3:1.9.4-SNAPSHOT
+google-cloud-core:2.13.0:2.14.0-SNAPSHOT
+google-cloud-shared-dependencies:3.5.0:3.6.0-SNAPSHOT
```

Comment:
They are from https://github.com/googleapis/google-cloud-java/blob/e7d648dd992a59df16cbdeb47623511f5ebb6d19/versions.txt#L606

- The latest google-cloud-core is 2.13.0 https://central.sonatype.com/artifact/com.google.cloud/google-cloud-core/2.13.0
- The latest shared dependencies BOM is 3.5.0 https://central.sonatype.com/artifact/com.google.cloud/google-cloud-shared-dependencies/3.5.0